certified-connectors/Cognito Forms/readme.md

```diff
@@ -1,21 +1,12 @@
-## Cognito Forms
+# Cognito Forms
 Cognito Forms lets you easily build powerful online forms, such as surveys, order forms, registration forms and more. For free.
 Please see our [support documentation](https://www.cognitoforms.com/support/63/data-integration/microsoft-power-automate) 
 or [contact us](https://www.cognitoforms.com/supportrequest/) for help with our public Power Automate connector.
 
-## Building the connector 
```

Comment:
I moved this to the bottom to put things in the same order as the template. I also renamed it to `Deployment Instructions`.